sirepo/auth_db.py

```diff
@@ -61,8 +64,8 @@ def audit_proprietary_lib_files():
 
     def _add(proprietary_code_dir, sim_type, sim_data_class):
         p = proprietary_code_dir.join(sim_data_class.proprietary_code_tarball())
-        sirepo.simulation_db.verify_app_directory(sim_type)
-        with sirepo.simulation_db.tmp_dir(chdir=True) as t:
+        sirepo.simulation_db.verify_app_directory(sim_type, uid=uid)
```

Comment:
This really shouldn't be here. We should have a registration interface so auth_db doesn't have to import all this stuff, which is irrelevant to the service that auth_db provides.